doc/progress-indicator.md

```diff
@@ -1,7 +1,7 @@
 # Progress Indicator
 
 ### What is it?
-A user interface element that indicates the progress of an operation.
+The progress indicator is a component that informs users about the status of ongoing processes, such as loading an application, submitting a form, or saving updates.
```

Comment:
such as loading **_an_** application. Nessa parte em destaque não deveria ser "and"?

Reply:
nesse caso o an está sendo usado como _uma_: "como carregar uma aplicação.."